woeip/apps/air_quality/dustrak.py

```diff
@@ -115,9 +115,10 @@ def load_dustrak(contents, tz):
     contents = io.StringIO(contents)
 
     header = {}
-    lines = itertools.takewhile(lambda x: x != '\n', contents)
+    lines = itertools.takewhile(lambda x: x != '\r\n', contents)
     for line in lines:
-        line = line.rstrip('\n')
+        print(line)
```

Comment:
`print` is not useful in a production system. If this is important, use `logger.info` to ensure log lines are written to console, files, wherever else logs are routed.